src/index.js

```diff
@@ -1,30 +1,57 @@
 import './style.css';
 import {
-  renderTasks, addItem, removeItem, toggleItem, clearCompleted,
+  renderTasks, addItem, toggleItem, clearCompleted, updateItemDescription, removeItem,
 } from './todo.js';
 
-const addItemButton = document.getElementById('addItemButton');
+const newItemInput = document.getElementById('newItemInput');
 const taskList = document.getElementById('taskList');
 const clearCompletedButton = document.getElementById('clearCompletedButton');
 
-addItemButton.addEventListener('click', handleAddItem);
+newItemInput.addEventListener('keydown', handleAddItem);
 taskList.addEventListener('click', handleTaskClick);
 clearCompletedButton.addEventListener('click', handleClearCompleted);
 
-function handleAddItem() {
-  addItem();
+function handleAddItem(event) {
+  if (event.key === 'Enter') {
+    addItem();
+  }
 }
```

Comment:
- [ ] *IMPORTANT NOTE: Please let's always read all requirements before we start building our project.* 

- [ ] It is important to follow the project core requirements, This will display the level of our attention to detail which is super important for software developers. 🛠💪

- [ ] As requested by the previous reviewer, In this project, you will build a simple HTML list of To-Do tasks

### [Project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md)
- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Create an **index.html** file and write your HTML markup here. Create an empty **To Do List** placeholder (`<div>` or `<ul>` element). The **index.html** file must be set as a template using the *HTML Webpack Plugin*.
- Create an **index.js** file and set an array of some simple *to do tasks* (array of objects). Each task object should contain three keys: 
  1. description [`string`].
  2. completed [`bool`].
  3. index: [`number`].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the `index` values for each task.
- Create a **style.css** and set rules for the **To Do List**. CSS must be loaded by *Webpack Style/CSS Loader*. Your list should be a clone of the part of the minimalist project captured in the video below.

- [ ] All other functionalities will be implemented in the later milestones 😀

Reply:
I think, that this time I get it!!!

![image](https://github.com/lerfast/To-do-list-webpack/assets/111823606/11dd6d95-4cf0-4c2f-8e80-9d08316272d5)